<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LobbyWindow</class>
 <widget class="QDialog" name="LobbyWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>451</width>
    <height>378</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(252, 240, 255);</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>290</width>
     <height>221</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLabel" name="label_ip">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>IP Address</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_name">
             <property name="text">
              <string>Player Name</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_port">
             <property name="text">
              <string>Port Number</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QListWidget" name="listWidget">
           <item>
            <property name="text">
             <string>player1</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>player2</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>player3</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>player4</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>player5</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>player6</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>280</y>
     <width>102</width>
     <height>66</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QPushButton" name="buttonStartGame">
      <property name="text">
       <string>Start Game</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="buttonExit">
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>431</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="sourceMainWindow.qrc">:/plots/monopoly_title.jpeg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>140</y>
     <width>111</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="sourceMainWindow.qrc">:/plots/monopoly_welcome.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="sourceMainWindow.qrc"/>
 </resources>
 <connections/>
</ui>
